demo-8-cache.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
-set -xeuo pipefail
 IFS=$'\n\t'
 
 . ./includes/setup.inc.sh
 
+set -euox pipefail
```

Comment:
That is on top for a purpose. Now you wont see errors by setup.inc.sh.

Reply:
Yes, I understand, and that's why I moved it down a few lines.

The 'set -x' was there so our audience can see what we demo. We don't them to see each line of the magic happening in **setup.inc.sh**, because that would distract their focus from our actual demo.